desktop/desktop_panels.xml

```diff
@@ -296,26 +249,22 @@
 					<left anchor="left" offset="0" />
 					<top anchor="bottom" offset="10" />
 				</anchored>
-				<!-- <font>calendarbold</font> -->
 				<min>0</min>
 				<tooltip textres="desktop_addday_tooltip" />
 				<script>
 					function onInit()
-						DB.setValue(getDatabaseNode(), "addday", "number", (getValue() or 0));
+						setValue(tonumber(OptionsManager.getOption(TimeManager.CLOCKADJUSTER_DEFAULT_DAYS)) or 0);
```

Comment:
Initialize with the value from the options.  Done for all of these add controls and short/long rest.